extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -166,3 +166,56 @@
   text-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
   white-space: nowrap !important;
 }
+
+/** For amp-story-inline-page-attachment-v2 experiment elements. */
+.i-amphtml-story-inline-page-attachment-chip {
+  --cta-chip-bg-color: rgba(255, 255, 255, 0.5);
+  background-color: var(--cta-chip-bg-color) !important;
```

Comment:
Let's save the variables for the theming PR actually. 
That way we can see it all working together with the theme attribute.

Reply:
yeah I agree with that